web/app/src/ui/board/GameBoardRow.tsx

```diff
@@ -0,0 +1,21 @@
+import {Game} from "../../model/game";
+import React from "react";
+import {GameBoardCell} from "./GameBoardCell";
+
+/**
+ * Game board row of cells.
+ */
```

Comment:
useless comment. You're just paraphrasing the function name. If you choose a good name (and you should), there is no need for an additional comment.